.gitignore

```diff
@@ -31,3 +31,4 @@
 /perf*
 .DS_Store
 .idea/*
+.vscode/*
```

Comment:
These should be in your personal config

Reply:
I think we don't want to store `.vscode/*` from anybody, not just from me.

Reply:
It's old project policy essentially. Hasn't come up in quite a long time surprisingly (other than https://github.com/JuliaLang/julia/pull/34169 sneaking .idea in?), per https://github.com/JuliaLang/julia/commit/f62ac13b42f11ca128117e77fab1632f7e9ac50c#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947

Reply:
I think we're going to need to adjust that policy in the near future; there are too many VSCode extensions that will store state that other projects are going to want to embed in their `.vscode` folders that devs aren't going to want to put this in their global `.gitignore` files.

Reply:
I'll remove this from this PR though, it's totally unrelated.